plugin/pkg/admission/persistentvolume/label/admission.go

```diff
@@ -57,6 +58,10 @@ func NewPersistentVolumeLabel() *persistentVolumeLabel {
 	}
 }
 
+func (l *persistentVolumeLabel) SetCloudConfig(cloudConfig io.Reader) {
```

Comment:
It does run without it. For AWS  relevant cloud configuration is either picked from environment variable or in case of GCE - instance must have API access.